ee/clickhouse/queries/trends/breakdown.py

```diff
@@ -91,7 +91,15 @@ def _format_breakdown_query(self, entity: Entity, filter: Filter, team_id: int)
             )
 
         if len(_params["values"]) == 0:
-            return "SELECT 1", {}, lambda _: []
+            return (
+                """
+                SELECT [now()] AS date, [0] AS data, '' AS breakdown_value LIMIT 0
+                /* optimized empty broken down subquery due to there being
```

Comment:
I don't get this comment?

Also, worth moving the comment from sql to python?

Reply:
It's specifically in SQL so that the hypothetical bugfixingperson seeing the SQL in Sentry/`pytest` output has an idea of why they see this slightly odd query. I think the additional few bytes over the air _may_ be worth it, having had to debug some special pieces of SQL recently and `ripgrep`ping things to then figure out what purpose they serve. WDYT?

Reply:
Problem with comments-in-sql is that they compose in weird ways - e.g. looking at query logs it's not easy to figure out where a funnel comment is coming from or if it's valid.

I'll need to anyways:
1. Reformat the SQL to make sense of it
2. Find the bit that's broken (non-trivial)

In my case I always then look up the actual code, rather than try to reverse-engineer the SQL. Is that not the case for you?

In either case:

If we wanted to leave this in SQL, how about:
1. Move this comment _before_ the select (as comments usually precede a code block).
2. Sentence casing
3. Improve wording to be more clear, since after this change nothing will be broken so. Suggestion: `Return 0 when no relevant events in this breakdown subquery`

If we go for not-in-sql, 2 and 3 still apply.

Reply:
Yeah, actually let's not bloat SQL with this – moved the comment to Python only and made it much more useful while at it (at this point I think it's just too long to be sensible in SQL).